src/bindings/js/node/tests/model.test.js

```diff
@@ -111,4 +111,33 @@ describe('Model.getOutputSize()', () => {
   it('should return 1 for the default model', () => {
     assert.strictEqual(model.getOutputSize(), 1, 'Expected getOutputSize to return 1 for the default model');
   });
+});
+
+describe('Node.js Model.getOps()', () => {
+  it('should return array of Node operations', () => {
+    const model = new addon.ModelWrap();
```

Comment:
You cannot get `Model` instance using constructor. `ModelWrap` doesn't have constructor. You can get it by call `Core.readModel` method.